src/test/compile-fail/panic-runtime/abort-link-to-unwind-dylib.rs

```diff
@@ -0,0 +1,15 @@
+// Copyright 2016 The Rust Project Developers. See the COPYRIGHT
+// file at the top-level directory of this distribution and at
+// http://rust-lang.org/COPYRIGHT.
+//
+// Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+// http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+// <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your
+// option. This file may not be copied, modified, or distributed
+// except according to those terms.
+
+// compile-flags:-C panic=abort -C prefer-dynamic
+// error-pattern:`panic_unwind` is not compiled with this crate's panic strategy
```

Comment:
can you add a comment here about what the purpose of this test is?

I am guessing that what is happening has to do with libstd, but it's not at all obvious to me.